src/Dfe.PlanTech.DatabaseUpgrader/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using DbUp;
+using System.Reflection;
+
+/// <summary>
+/// PlanTech Database Upgrader.
+/// </summary>
+internal class Program
+{
+    const int ERROR_RESULT = -1;
+    const int SUCCESS_RESULT = 0;
+
+    private static int Main(string[] args)
+    {
+        if (args == null || !args.Any())
+        {
+            DisplayError("Please supply a connection string.");
```

Comment:
Might be worth returning error result here, instead of continuing?

Reply:
good spot

Reply:
Updated this now